sources/reference/admiral-vault.md

```diff
@@ -0,0 +1,36 @@
+page_main_title: Shippable installer
+main_section: Reference
+sub_section: Admiral
+page_title: Admiral - Vault Configuration
+page_description: Admiral Vault Configuration
+page_keywords: install, microservices, Continuous Integration, Continuous Deployment, CI/CD, testing, automation, pipelines, docker, lxc, vault
+
+# Admiral Vault Configuration
+After running the CLI portion of the Admiral installer, you can visit the UI at the provdied URL with the provided token to continue setting up Shippable.
+
+## Admiral UI
+When you first log in to the Admiral UI, Vault will not yet be installed or initialized.  Shippable gives you the opportunity to decide if you'd like to install on a single box, connect to an existing instance of Vault, or install a brand new copy of vault on a separate machine.  This page will discuss installing a new copy on a separate machine.
+
+## Configuration
+
+Once you're in the Admiral UI, you should see a table of options in the "intiialize infrastructure" section.  After the database, there is a section called "secrets".  Vault is the encryption provider that Shippable utilizes, so this is the section we will modify to get Vault set up on a new machine.  Shippable utilizes version `v0.6.0` of vault.
+
+Select the "New Node" radio button, enter the IP address of your machine, and run the command that is displayed.  Once the command is run, check the box beneath it so that we know you've made it far enough in the workflow to proceed with the initialization.
+
+<img src="../../images/reference/admiral/admiral-vault-ui-setup.png" alt="Vault machine setup">
+
+## Installation
+
+You may need to fill in values for the other services listed on the page inside the "initialize infrastructure" section (messaging, state, redis, swarm).  Much like Vault, these services can also be brought up during the onebox install, or on their own machines.
```

Comment:
I'm not sure about "during the onebox install."  It's the same installation series for components on the same machine and other machines, in the same order.  Just different scripts depending on the location.